docs/modules/preseq.md

```diff
@@ -19,8 +19,45 @@ back the x axis until one of the datasets shows 90% of its maximum y-value
 (unique molecules).
 
 To disable this feature and show all of the data, add the following to your
-MultiQC configuration:
+[MultiQC configuration](http://multiqc.info/docs/#configuring-multiqc):
 ```yaml
 preseq:
     notrim: true
 ```
+
+#### Using coverage instead of read counts
+
+Preseq reports its numbers as "Molecule counts". This isn't always very intuitive, 
+and it's often easier to talk about sequencing depth in terms of coverage.
+
+You can plot with approximate coverage on the axes instead by specifying the 
+reference genome or target size, and the read length in your 
+[MultiQC configuration](http://multiqc.info/docs/#configuring-multiqc):
+```yaml
+preseq:
+    genome_size: 3049315783
+    read_length: 300
+```
+These parameters make the script take every molecule count and divide it by 
+(genome_size / read_length).
+
+MultiQC comes with effective genome size presets for Human and Mouse, so you can 
+provide the genome build name instead, like this: `genome_size: hg38`. Only the
+following builds are supported: `hg19`, `hg38`, `mm10`.
```

Comment:
Depends where you are - nearly all of our preseq runs are on WGS :) But yes, this would be in addition to other more specific things that people could set. And as you say, I envisage more "add-on" features being added in the future which could find this useful. That was kind of my thinking - better to add a generic option like this earlier rather than retrofit later.